modules/preferences/test/spec.coffee

```diff
@@ -54,7 +54,7 @@ describe 'hx-preferences', ->
 
     describe 'timezone', ->
       it 'setter/getter', ->
-        hx.preferences.timezone('UTC+01:00').should.equal(hx.preferences)
```

Comment:
This was doing the correct thing before - most of the apis in hexagon return themselves when setting so that you can chain like this:

```
hx.preferences
  .timezone('UTC+1:00')
  .locale('bg')
```